admin/tests/integration/components/organizations/places_test.js

```diff
@@ -0,0 +1,74 @@
+import { module, test } from 'qunit';
+import { render } from '@1024pix/ember-testing-library';
+import { setupRenderingTest } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | Organizations | places', function (hooks) {
+  setupRenderingTest(hooks);
+
+  let store;
+
+  hooks.beforeEach(async function () {
+    store = this.owner.lookup('service:store');
+  });
+
+  module('Display places', function () {
+    test('it should display title and no results text', async function (assert) {
```

Comment:
**suggestion. :** Je rajouterais un petit module "when there is no results" ou alors je rajouterais ça dans la description dutest